client/ionic/package.json

```diff
@@ -31,11 +31,13 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "npm run lint && react-scripts test",
```

Comment:
Rather than running this, it might just be good to set the `CI=true` environment variable when running `react-scripts test` -- react-scripts will then automatically treat eslint warnings as errors (see the change I tried in #168)

Reply:
OK, that makes sense.

Reply:
@epicfaace to your comment on #168, did you ever figure out where the different eslint configs were coming from? definitely doesn't seem right for there to be 2 configs that could confuse devs when run at different times in the build process...

Reply:
Yes, we discussed it further here -- https://github.com/WorldHealthOrganization/app/issues/180. We probably shouldn't merge this PR as-is because as you said, it would cause two configs to be there.